web/modules/custom/configform/config/schema/configform.schema.yml

```diff
@@ -0,0 +1,26 @@
+# Schema for configuration files of the Configform module.
+
+configform.settings:
+  type: config_object
+  mapping:
+    name:
+      type: varchar
```

Comment:
I came across this thread 
[https://stackoverflow.com/questions/41136136/what-is-the-difference-between-varchar-and-string-in-sqlalchemys-data-type](url)

Since i am having length constraints on my fields that's why i used varchar here

Reply:
"sqlalchemy's data type"?
But this is Drupal no?
I don't think they are related.

Reply:
Ok i've changed them